### DISCUSSION TITLE
How do I disable log

### DISCUSSION THREAD

**Discussion by liuliang-wt** [elsa-workflows/elsa-core#4656] [Q&A]
I use a timer activity which runs every second, it will make the workflowinstance table very big... how do I make this kind of workflows run silently, without writing records into the workflowinstance table

**Comment by sfmskywalker** [comment]
Elsa 2 has a retention module that you can enable, which will delete workflow instances and execution logs when workflow instances complete. There are some options here that allow you to control things like max age of finished workflows, which you could set to e.g. 1 minute or less. I think there's even the option to delete the workflow execution logs immediately.

Elsa 3 doesn't have a retention module yet, but it does let you control a different storage provider for logs, including your own "noop" (no operation" provider that simply doe nothing.

**Reply by kelvenray** [reply]
Hi @sfmskywalker May I know more detail about this retention module?How to configurate it?
